# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/xfree86/imake_tmpl.patch
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./config/cf/Imake.tmpl.orig	Sun Mar 10 22:44:13 2002
+++ ./config/cf/Imake.tmpl	Sun Mar 10 22:44:52 2002
@@ -1036,10 +1036,10 @@
 #define GzipCmd gzip
 #endif
 #ifndef CppCmd
-#define CppCmd /LibDirName/cpp
+#define CppCmd cpp
 #endif
 #ifndef RawCppCmd
-#define RawCppCmd CppCmd -undef
+#define RawCppCmd CppCmd
 #endif
 #ifndef CppNoLineInfoOption
 #define CppNoLineInfoOption /**/
@@ -1491,6 +1491,7 @@
 
           INCROOT = IncRoot		/* base of where to put header files */
         USRLIBDIR = UsrLibDir		/* nonshared libraries */
+        MODULEDIR = $(USRLIBDIR)/modules /* Added by Clifford */
            VARDIR = VarDirectory	/* usually /var */
         VARLIBDIR = VarLibDir		/* xdm runtime files */
   SYSTEMUSRLIBDIR = SystemUsrLibDir	/* system's "/usr/lib" */
